sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -133,10 +133,12 @@ class PerElement(ptransform.PTransform):
 
     def expand(self, pcoll):
       paired_with_void_type = typehints.Tuple[pcoll.element_type, Any]
+      output_type = typehints.KV[pcoll.element_type, int]
```

Comment:
Is the change here still needed (just to confirm) ?

Reply:
Yes, it is. Cross-language `PerElement` doesn't work without explicit output type.